examples/Cargo.toml

```diff
@@ -114,6 +115,21 @@ path = "examples/z_get.rs"
 name = "z_forward"
 path = "examples/z_forward.rs"
 
+[[example]]
```

Comment:
I don't see in this PR any modification to the `README` nor comments in the code that explain how to uses these examples. I.e., difference between `z_liveniness`, `z_sub_liveness`, and `z_get_liveness`.